src/schema_lexer.xrl

```diff
@@ -6,6 +6,8 @@ STRING          = [a-zA-Z0-9_\.]+
 BOOL            = (true|false)
 WS              = [\s\t]+
 NL              = [\n\r]+
+COMMENT         = //.[^\n]*\n
```

Comment:
could be simplified to COMMENT = //[^\n]*

Reply:
@an2deg your recommended change doesn't fail with this style of comments:
```
//
// asdf
//
```